modules/mvt/test/data/fetch_data.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/local/bin/python3
```

Comment:
Does this run on CI?

Reply:
Also nit: `#!/usr/bin/env python3` is more portable.

Reply:
No this is not meant for CI, more to help me easily check that the new code is equivalent to the old. @kylebarron fixed